src/main/resources/sql/hw16_fetch.sql

```diff
@@ -0,0 +1,19 @@
+SELECT * FROM Homework;
+
+SELECT Lesson.name AS lesson_name, Lesson.updatedat, Homework.name AS hw_name, Homework.description 
+FROM Lesson JOIN Homework ON Lesson.homework_id = Homework.id;
+
+SELECT Lesson.name AS lesson_name, Lesson.updatedat, Homework.name AS hw_name, Homework.description 
+FROM Lesson JOIN Homework ON Lesson.homework_id = Homework.id ORDER BY Lesson.updatedat;
+
+SELECT Schedule.name AS sch_name, Schedule.updatedat, Lesson.name AS l_name, Lesson.updatedat
+FROM Schedule 
```

Comment:
It is good practice to use short aliases for table names.
In my opinion, schedule_name is better than sch_name and lesson_name is better than l_name because when using these columns it will be easier to understand what they contain.